modular_skyrat/modules/customization/modules/client/loadout/suit.dm

```diff
@@ -131,6 +131,36 @@
 	name = "Flak Jacket"
 	path = /obj/item/clothing/suit/flakjack
 	cost = 2
+	
+/datum/loadout_item/suit/job/security_jacket
+	name = "Security Jacket"
+	path = /obj/item/clothing/suit/toggle/jacket/sec
+	/* restricted_roles = list("Head of Security", "Security Officer", "Warden", "Detective", "Security Medic", "Security Sergeant")
```

Comment:
Can just delete this rather than commenting it out tbh

Reply:
Was told commenting out was preferred, may as well keep it

Reply:
fair